### PR TITLE
Add k8s-node v1.21 image.

### PR DESCRIPTION
**Pull request by ctalledo**
Signed-off-by: Cesar Talledo <ctalledo@nestybox.com>

### CODE REVIEW

File: k8s-node/1.21/kube-pull.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+dockerd > /var/log/dockerd.log 2>&1 &
+dockerd_pid=$!
+sleep 2
+
+# pull inner images
+kubeadm config images pull --kubernetes-version=$k8s_version
+# flannel cni
+docker image pull quay.io/coreos/flannel:v0.12.0-amd64
+# weaveNet cni
+docker image pull docker.io/weaveworks/weave-kube:2.8.1
```
Comment (rodnymolina) [inline k8s-node/1.21/kube-pull.sh:31]: Did you check that these are the exact images fetched by k8s v1.21? I'm asking coz I'm seeing that they match what we had in v1.20...

**Review by rodnymolina — COMMENTED** [review]
Check question pls.

File: k8s-node/1.21/kube-pull.sh (hunk 1 above)
Comment (ctalledo) [reply k8s-node/1.21/kube-pull.sh:31]: Yes, I just double checked; those are the latest and and they work fine with v1.21.12.

**Review by rodnymolina — APPROVED** [review]
LG

**Pull request merged**